playbooks/conferences.md

```diff
@@ -24,6 +24,30 @@ guidelines:
 - If you need help preparing a conference talk, stop by [Ash][ash]'s weekly Writing Office Hours.
 - It's also recommended sharing the talk and lessons with the team.
 
+## Approval Process
+
+- Find an event (conference/seminar/course) that you would like to attend. Identify items in the agenda that will
+  contribute to both your growth goals and Artsy's business goals.
+- Estimate any travel or lodging expenses that may be needed to attend the event using Concur to search for airfare
+  and hotels.
+- Speak to your manager about attending the event before purchasing tickets. Communicate the following items:
+  - How will this event contribute to your and your team's goals?
+  - What are the dates of the event?
+  - What is the attendance fee for the event?
+  - What travel and lodging feens will be required?
+  - Managers should consider the following:
+    - Does the event align with individual and business goals?
+    - Does the attendance cost fit within the remaining budget?
+    - Is attendance being requested far enough in advance to keep costs low?
+    - Have enough engineers being given a chance to attend events?
+- Purchase tickets for the event, and start an expense report in Concur or Vistra.
```

Comment:
Vistra?

Reply:
Vistra is the expensing platform that is used by UK and EU team members. I'm still not sure whether those team members will be using Concur to book travel and hotel accommodations _in addition to_ using Vistra for expensing tickets to the conference.

Reply:
Got it 👍